### ISSUE TITLE
Cleaning up grpauto.tst and other tests

### ISSUE THREAD

**Issue by alex-konovalov** [gap-system/gap#964]
Nightly tests in the master branch are failing after #896. I am creating new issue to continue discussion here instead of commenting on a merged PR:
- [x] terminate the test when GAP workspace reaches 2g : done in PR #959 
- [x] fix `testmanuals` with default packages : done in https://github.com/gap-system/gap/commit/5ca402f9c879a5db7f4c77a2b60c186d1f4f8c32
- [x] get rid of `#I  Alternating recognition needed!` messages when `testinstall/teststandard` run with no packages
- [x] make `grpauto.tst` run by simplifying tests or moving hard cases to benchmarks or other tests: #965, https://github.com/gap-system/gap/commit/cf1c2fbcfce9809c0c08546c180c09bdd665438a, https://github.com/gap-system/gap/commit/dcaa4b2bc8e10979421f953f01cd1803f4c545f6

**Comment by hungaborhorvath** [comment]
In #763 I manually changed the ````InfoLevel```` of ````InfoWarning```` in the test file to make sure to not get this warning message. It was two extra lines and it took care of the ````Alternating recognition needed```` message.

I am wondering, though, would it be possible to have an automatic and visible test for _without_ packages for a PR? Then one could see immediately if some of the code breaks tests without the packages. Before I never even thought about checking my code without packages, because I thought travis is taking care of all tests. (And btw, why exactly do we test for without packages, if GAP is shipped with some default packages, anyway?)

**Comment by alex-konovalov** [comment]
@hungaborhorvath : I've looked at the line which prints `Alternating recognition needed` and I can't see a reason why it uses `InfoWarning` while the rest of the info messages in this file are using `InfoLattice`, so I've changes it to `InfoLattice` to in https://github.com/gap-system/gap/commit/488af13d9a23e7c5b6ae6acffe8275b4d59cfac9 - now testinstall should pass.

Testing GAP with/without packages is important due to complicated dependencies and not all packages working everywhere. There are no fundamental obstacles to do it as you suggest on Travis - the limitation of Travis tests is due to its restricted resources, and our willing for a faster turnover: you will have to wait twice longer for the test to complete if you will run each configuration with/without packages on Travis...

**Comment by alex-konovalov** [comment]
@hungaborhorvath P.S. and of course we need a way to check that packages does not break the behaviour of the core system, so we need a reference run of the tests without packages.

**Comment by alex-konovalov** [comment]
I have removed some more heavy tests in https://github.com/gap-system/gap/commit/cf1c2fbcfce9809c0c08546c180c09bdd665438a, https://github.com/gap-system/gap/commit/dcaa4b2bc8e10979421f953f01cd1803f4c545f6 - now `grpauto.tst` is runnable, and it takes about 2000 seconds both with and without packages. 

This means that we're able to merge and test further PRs, and I am going to merge #763 by @hungaborhorvath now. 

However, the "alternating recognition message" is still there when GAP is started with `-A` and I would appreciate if @hulpke could say whether it's OK to show it with `InfoPerformance` level 2.

I've removed the dependency of this issue on benchmarks, as those may be independently continued in #966.

**Comment by hulpke** [comment]
I'm not fighting for the level of `InfoPerformance`, but the whole idea of this InfoClass was to warn if something carelessly inefficient is happening. I find it somewhat strange that we are on the path of having the test files dictate this user interface.

My recommendation would be to rather have the tests deliberately turn InfoPerformance (maybe even InfoWarning) to zero to ensure they do have to bother with such information.

**Comment by alex-konovalov** [comment]
@hulpke - we have mechanism for that: I suggested already to use START_TEST and STOP_TEST to play with InfoLevel elsewhere, but you haven't commented on that. For now, only `InfoPerformance` will suffice.

**Comment by alex-konovalov** [comment]
`InfoPerformance` now handled in https://github.com/gap-system/gap/commit/392ad821fd45b2b5710933cf6dc77ae1007f3460
